benchbuild/utils/run.py

```diff
@@ -10,9 +10,14 @@
 
 from benchbuild import settings, signals
 
+if t.TYPE_CHECKING:
+    import mypy_extensions as mypy  # pylint: disable=unused-import
```

Comment:
Codacy found an issue: [Unable to import 'mypy_extensions'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)